src/components/date/__internal__/navbar/navbar.spec.tsx

```diff
@@ -1,12 +1,15 @@
 import React from "react";
 import TestRenderer from "react-test-renderer";
-import { shallow } from "enzyme";
+import { shallow, ShallowWrapper } from "enzyme";
 
-import Navbar from "./navbar.component";
+import Navbar, { NavbarProps } from "./navbar.component";
 import StyledButton from "./button.style";
 
 describe("Navbar", () => {
-  let wrapper, onPreviousClick, onNextClick;
+  let wrapper: ShallowWrapper;
+  let onPreviousClick: null | jest.Mock;
```

Comment:
```suggestion
  let onPreviousClick: jest.Mock;
```

**suggestion** (non-blocking): I see no reason for the `null` option here or for `onNextClick`. If we remove that the `?.` before checking the `mock` property can go.